src/content/developers/docs/consensus-mechanisms/index.md

```diff
@@ -5,59 +5,57 @@ lang: en
+These components together form the consensus mechanism.
 
 ## Types of consensus mechanisms {#types-of-consensus-mechanisms}
 
-### Proof-of-work {#proof-of-work}
+### Proof-of-work based {#proof-of-work}
 
-Ethereum, like Bitcoin, currently uses a **proof-of-work (PoW)** consensus protocol.
+Like Bitcoin, Ethereum once used a **proof-of-work (PoW)** based consensus protocol.
 
 #### Block creation {#pow-block-creation}
 
-Proof-of-work is done by [miners](/developers/docs/consensus-mechanisms/pow/mining/), who compete to create new blocks full of processed transactions. The winner shares the new block with the rest of the network and earns some freshly minted ETH. The race is won by the computer which is able to solve a math puzzle fastest – this produces the cryptographic link between the current block and the block that went before. Solving this puzzle is the work in "proof-of-work".
+Proof-of-work is done by [miners](/developers/docs/consensus-mechanisms/pow/mining/), who compete to create new blocks full of processed transactions. The winner shares the new block with the rest of the network and earns some freshly minted ETH. The race is won by the computer which is able to solve a math puzzle fastest—this produces the cryptographic link between the current block and the block that went before. Solving this puzzle is the work in "proof-of-work". The true head of the chain is then determined by a fork-choice rule that selects the set of blocks that have had the most work done to mine them.
 
 #### Security {#pow-security}
 
 The network is kept secure by the fact that you'd need 51% of the network's computing power to defraud the chain. This would require such huge investments in equipment and energy; you're likely to spend more than you'd gain.
 
 More on [proof-of-work](/developers/docs/consensus-mechanisms/pow/)
 
-### Proof-of-stake {#proof-of-stake}
+### Proof-of-stake based {#proof-of-stake}
 
-Ethereum plans to upgrade to a **proof-of-stake (PoS)** consensus protocol.
+Ethereum now uses a **proof-of-stake (PoS)** based consensus protocol.
 
 #### Block creation {#pos-block-creation}
 
 Proof-of-stake is done by validators who have staked ETH to participate in the system. A validator is chosen at random to create new blocks, share them with the network and earn rewards. Instead of needing to do intense computational work, you simply need to have staked your ETH in the network. This is what incentivises healthy network behaviour.
 
 #### Security {#pos-security}
 
-A proof-of-stake system is kept secure by the fact that you'd need 51% of the total staked ETH to defraud the chain. And that your stake is slashed for malicious behaviour.
+A proof-of-stake system is kept secure crypto-economically because a huge amount of ether must be destroyed by an attacker attempting to take control of the chain. A system of rewards incentivizes individual stakers to behave honestly, and penalties disincentivize stakers from acting maliciously.
```

Comment:
```suggestion
A proof-of-stake system is secure crypto-economically because an attacker attempting to take control of the chain must destroy a massive amount of ETH. A system of rewards incentivizes individual stakers to behave honestly, and penalties disincentivize stakers from acting maliciously.
```